File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.*;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		StringTokenizer st = new StringTokenizer(sc.nextLine());
		int N = Integer.parseInt(st.nextToken());
		int M = Integer.parseInt(st.nextToken());

		int[] basket = new int[N];

		for (int i = 0; i < M; i++) {
			st = new StringTokenizer(sc.nextLine());
			int min = Integer.parseInt(st.nextToken());
			int max = Integer.parseInt(st.nextToken());
			int ball = Integer.parseInt(st.nextToken());

			for (int j = min - 1; j < max; j++) {
				basket[j] = ball;
			}
		}

		sc.close();

		String output = "";

		for (int i = 0; i < basket.length; i++) {
			output += basket[i] + " ";
		}

		System.out.println(output);
	}
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
N, M = map(int, input().split())

basket = []
for _ in range(N):
    basket.append(0)

for _ in range(M):
    min, max, ball = map(int, input().split())
    
    for i in range(min-1, max):
        basket[i] = ball;

print(*basket)